src/libs/CollectionUtils.ts

```diff
@@ -0,0 +1,10 @@
+function lastItem<T>(object: Record<string, T> = {}): T {
+    const lastKey = Object.keys(object).pop() ?? 0;
+    return object[lastKey];
+}
+
+function extractCollectionItemID(key: string) {
```

Comment:
```suggestion
/**
 * Used to grab the id for a particular collection item's key.
 * e.g. reportActions_1 -> 1
 */
function extractCollectionItemID(key: `${OnyxCollectionKey}${string}`) {
```

Leave JSDoc comments.

And we can type `key` stricter.